### ISSUE TITLE
Executable not working

### ISSUE THREAD

**Issue by Teg3z** [Teg3z/Wheel-of-Luck#21]
I found out that creating an executable of wheel_of_luck.py by installer.py is working, although the executable isn't working at all.

So we should fix that, or remove the installer.py completely and setup the creation of an executable via GH Actions to move into CI/CD?

**Comment by R4tmax** [comment]
No, screw then installer, we can try to provide .msi, .zip or .exe files via GitHub ecosystem, there are some issues that worry me a bit (eg. how will the configurations work when they are pulled via external artefact) 

I think we need to double check if all pathfiles are relative, and perhaps it would be enough to just notify the users on how to organize the folder. 
I believe it should be possible to use custom scripts within the actions ecosystem, so there should be a lot of options how to achieve this.

**Comment by R4tmax** [comment]
Adressed partially by #24 , take note that testing IS required on both the Workflow execution and utilization of the final packages.

**Comment by Teg3z** [comment]
Ok with #24 this issue is irrelevant, if the executable wont be working after #24, we raise new issue.
